example/src/main/java/com/thefuntasty/mvvmsample/domain/dummy/SaveDataToFirstServerUseCase.kt

```diff
@@ -0,0 +1,14 @@
+package com.thefuntasty.mvvmsample.domain.dummy
+
+import com.thefuntasty.mvvm.crusecases.UseCase
+import com.thefuntasty.mvvmsample.tools.randomError
+import kotlinx.coroutines.delay
+import javax.inject.Inject
+
+class SaveDataToFirstServerUseCase @Inject constructor() : UseCase<String, String>() {
+    override suspend fun build(args: String): String {
+        delay(1000)
```

Comment:
<table data-meta="generated_by_danger-pr">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/1e22aaaf9def5bbeda391f2cd9735be2fc018fbd/example/src/main/java/com/thefuntasty/mvvmsample/domain/dummy/SaveDataToFirstServerUseCase.kt#L10"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>